boards/shields/esp_8266/boards/numaker_pfm_m467.overlay

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright (c) 2023 Nuvoton Technology Corporation.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+&pinctrl {
+    uart2_esp8266: uart2_esp8266 {
+	group0 {
+		pinmux = <PC1MFP_UART2_TXD 0x0000>,
```

Comment:
should be <PC1MFP_UART2_TXD> instead of <PC1MFP_UART2_TXD 0x0000>

Reply:
fixed pinmux format error